test/core/floatBE.test.js

```diff
@@ -0,0 +1,9 @@
+"use strict";
+
+describe("floatBE", function() {
+  it("reads a float, big-endian", function() {
+    var b = Buffer.from([1, 2, 3, 4]);
+    var p = Parsimmon.Binary.floatBE;
+    assert.deepEqual(p.tryParse(b), 2.387939260590663e-38);
```

Comment:
Believe it or not, this is my lucky number!!

Reply:
Lol, I got this directly from the node buffer page.  I figured if it's good enough for their example, it's good enough for a test case.